src/test/java/com/kobylynskyi/graphql/codegen/GraphQLCodegenAnnotationsTest.java

```diff
@@ -321,4 +322,14 @@ private void generate(String path) throws IOException {
                 TestUtils.getStaticGeneratedInfo(mappingConfig)).generate();
     }
 
+    @Test
+    void generate_CustomTemplates_Type() throws Exception {
```

Comment:
This test case is not related to annotations, so please move it to a separate test class `GraphQLCodegenCustomTemplatesTest`